sdccc/pom.xml

```diff
@@ -15,7 +15,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <junitVersion>5.6.0</junitVersion>
         <junitPlatformVersion>1.6.0</junitPlatformVersion>
-        <sdcriVersion>2.1.0-20221018.121941-19</sdcriVersion>
+        <!-- TODO: this must be a Release Version when building the Release -->
```

Comment:
add reference to https://github.com/Draegerwerk/SDCcc/issues/41 here

Reply:
Done.